src/classes/User.js

```diff
@@ -0,0 +1,41 @@
+class User {
+  constructor(userData = {}) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.pantry = userData.pantry;
+    this.recipesToCook = [];
+  }
+
+  addRecipesToCook(recipe) {
+    this.recipesToCook.push(recipe);
+  }
+  
+  removeRecipesToCook(recipe) {
+    let recipeToRemove = this.recipesToCook.indexOf(recipe);
+    this.recipesToCook.splice(recipeToRemove, 1);
```

Comment:
If `indexOf()` doesn't find a matching element, it returns `-1`. If `splice()` receives -1 as its second argument, will everything be ok with this implementation of the code?

from MDN:
> If deleteCount is 0 or negative, no elements are removed. In this case, you should specify at least one new element (see below).

Reply:
included a kind error message for now, will figure out a better solution once we display functionality on the DOM